code-challenges.js

```diff
@@ -37,18 +66,36 @@ var myString2 = "charlie"
 
 var numberExample1 = 42
 var numberExample2 = 27
+// create a function called subtraction that takes in two numbers.
+// const subtraction = (num1, num2) =>{
+// if two numbers are inputed, return the answer to the first one being subtracted by the second.
+//     return num1 - num2
+// }
+// console.log(subtraction(42, 27))
 
-var numberExample3 = 7
-var numberExample4 = 19
-
+// var numberExample3 = 7
+// var numberExample4 = 19
 
+// const subtraction = (num1, num2) =>{
+//     return num1 - num2
+// }
+// console.log(subtraction(42, 27))
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to also accept non-number edge cases. Inform your user if the variable passed into the subtraction function is not a number.
 // Use the two sets of test variables provided below. Expected output: "Your input is not a number", 3
 // Hint: There is an operator in JavaScript called typeof :)
 
 var numberExampleRefactor1 = 42
 var numberExampleRefactor2 = "hello"
-
-var numberExampleRefactor3 = 27
-var numberExampleRefactor4 = 24
+// tried including type of in different areas but I was not able to get it to work.
```

Comment:
What did that look like?

Reply:
The output returned NaN because I did not format my function correctly.